package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@artus/pipeline",
+  "version": "0.0.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "npm run build:dist",
+    "build:dist": "tsc -b ./tsconfig.json"
+  },
+  "dependencies": {
```

```suggestion
  "devDependencies": {
```

typescript 不要放在直接依赖里面吧